enzyme.config.js

```diff
@@ -0,0 +1,5 @@
+/* eslint-disable import/no-extraneous-dependencies */
+import Enzyme from 'enzyme';
+import Adapter from 'enzyme-adapter-react-16';
```

Comment:
Hello @el-Joft, Nice work so far, but it will be better if you can import all test configurations here and export them globally so that we can reuse them easily; for instance 
import Enzyme,{ shallow } from 'enzyme';
import configureStore from 'redux-mock-store';
import thunk from 'redux-thunk';

global.shallow = shallow;
global.configureStore = configureStore;
global.thunk = thunk;

We can reuse it like this =>
const wrapper = shallow(<App />);
const mockStore = configureMockStore([thunk]);

Also, check out how to setup jest-localstorage-mock to enable us test code that deals with windows local storage
https://www.npmjs.com/package/jest-localstorage-mock